package.json

```diff
@@ -35,10 +43,11 @@
     "webpack-dev-server": "^1.14.1"
   },
   "scripts": {
-    "test": "echo 'Yee-haw!'",
+    "testserver": "cross-env NODE_ENV=test karma start internals/testing/karma.conf.js",
```

Comment:
Use this command to keep a testing daemon running locally. Tests will be run every time a file is modified.